### ISSUE TITLE
 error TS2307: Cannot find module 

### ISSUE THREAD

**Issue by rlucky469** [ckeditor/ckeditor5#14205]
## 📝 Provide a description of requested docs changes

![image](https://github.com/ckeditor/ckeditor5/assets/77981064/1da02faa-e156-4d8c-96f4-8fe3cf93476f)

![image](https://github.com/ckeditor/ckeditor5/assets/77981064/27561008-87af-4232-87e6-3feb2601229b)

**Comment by Reinmar** [comment]
@oleq is debugging this – it seems that there's an import that works locally and on our CIs but not when used in a certain situations ;/.

**Comment by oleq** [comment]
This is what I found out so far:


[In document-outline/src/index.d.ts](https://npmview.vercel.app/@ckeditor/ckeditor5-document-outline) we have

```
export declare const icons: {
    tableOfContentsIcon: string;
};
```

as a result of the following index.ts

```
import tableOfContentsIcon from '../theme/icons/table-of-contents.svg';

export const icons = {
	tableOfContentsIcon
};
```

and this is fine.

---

[In ui/src/index.d.ts](https://npmview.vercel.app/@ckeditor/ckeditor5-ui) there is

```
export { default as ColorPaletteIcon } from '../theme/icons/color-palette.svg';
```

as a result of the following `index.ts`

```
export { default as ColorPaletteIcon } from '../theme/icons/color-palette.svg';
```

and that causes errors

**Comment by Witoso** [comment]
I just found that `"skipLibCheck": true` hides it for the `tsc` and the error is not produced if you're consuming the package. Not sure if it worked the same for our CI but we have this setting in our main `tsconfig`.

**Comment by rlucky469** [comment]
it worked thank you @Witoso

**Comment by Witoso** [comment]
While `"skipLibCheck": true` may be a workaround, having this configuration option set is not a good practice. We will keep this open to implement a proper fix.

**Comment by mlewand** [comment]
The fix has been merged. We plan to release a hotfix release tomorrow.

For those of you facing this issue, you can try [a workaround posted by @Witoso](https://github.com/ckeditor/ckeditor5/issues/14205#issuecomment-1557282578).

**Comment by mlewand** [comment]
This problem has been fixed with 38.0.1 hotfix. I'm unpinning this issue.

**Comment by rlucky469** [comment]
CKEditor 5 online builder
![image](https://github.com/ckeditor/ckeditor5/assets/77981064/3829a4e3-4c46-46f9-9e2f-e3fbfd5de798)
